games/c.yaml

```
@@ -120,6 +120,31 @@
   updated: 2020-02-08
   url: https://bitbucket.org/dalerank/caesaria/wiki/Home
 
+- name: Candy Crisis
+  originals:
+  - Puyo Puyo Series
```

```suggestion
  - Candy Crisis
```
this should be considered a source port (remake)